### PR TITLE
fix regexp

### LINKED ISSUE #152: (SP: 0.5 ) Implement the validation for all the fields on the form
UserStory: #79 

## Description
Implement the validation for all the fields on the sign-up form. 
## Design 

Image

## Acceptance Criteria
- [x] "First Name" and "Last Name" fields must receive a string in a range from 2 to 15 symbols.
- [x] The email field must be for emails.
- [x] Password must contain at least one alphabetic and one numeric character.
- [x] The "Confirm Password" field must be connected with the "Password" field.

### PR DESCRIPTION
**Pull request by Mygal-Alex**
close #152 
- [x] "First Name" and "Last Name" fields must receive a string in a range from 2 to 15 symbols.
- [x] The email field must be for emails.
- [x] Password must contain at least one alphabetic and one numeric character.
- [x] The "Confirm Password" field must be connected with the "Password" field.

### CODE REVIEW

**Comment by marianamartsy** [comment]
No description provided. Add some description, please

**Review by marianamartsy — APPROVED** [review]
👍

**Review by AndriiNasadiuk — APPROVED** [review]
👌

**Review by ArtemHolikov — APPROVED** [review]
👍

File: src/utils/validations/common.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const validations = {
   nameField: (value) => {
-    if (value.length > 30) {
+    if (value.length < 2 || value.length > 16) {
```
Comment (vlad-khrychov) [inline src/utils/validations/common.js:3]: I do not like "magic numbers"

**Review by vlad-khrychov — APPROVED** [review]

**Pull request merged**